definitions/npm/firebase_v4.x.x/flow_v0.46.x-/firebase_v4.x.x.js

```diff
@@ -0,0 +1,410 @@
+/* @flow */
+
+/**** firebase ****/
+
+declare interface FirebaseConfig {
+  apiKey: string;
+  authDomain?: string;
+  databaseURL: string;
+  projectId: string;
```

Comment:
Ya I realize that when looking at the firebase sdk their typescript does show the projectId as optional.